src/main/java/org/codehaus/groovy/ast/expr/RangeExpression.java

```diff
@@ -27,26 +27,42 @@
  * <pre>for i in 0..10 {...}</pre>
  */
 public class RangeExpression extends Expression {
```

Comment:
Is there any hope that code like AST transforms that depend on "from" and "to" and "inclusive" will be able to handle a left-open or full-open range literal?  Is it considered a breaking change to use "<" on the left?